src/current/v23.2/cockroach-debug-zip.md

```diff
@@ -111,7 +113,8 @@ Flag | Description
-`--include-range-info` | Include one file per node with information about the KV ranges stored on that node, in `nodes/{node ID}/ranges.json`.<br /><br />This information can be vital when debugging issues that involves the [KV storage layer]({% link {{ page.version.version }}/architecture/overview.md %}#layers), such as data placement, load balancing, performance or other behaviors. In certain situations, on large clusters with large numbers of ranges, these files can be omitted if and only if the issue being investigated is already known to be in another layer of the system (for example, an error message about an unsupported feature or incompatible value in a SQL schema change or statement). Note however many higher-level issues are ultimately related to the underlying KV storage layer described by these files so only set this to `false` if directed to do so by Cockroach Labs support.<br /><br />**Default:** true
+<a name="include-goroutine-stacks"></a>`--include-goroutine-stacks` | Fetch stack traces for all goroutines running on each targeted node in `nodes/*/stacks.txt` and `nodes/*/stacks_with_labels.txt` files. Note that fetching stack traces for all goroutines is a "stop-the-world" operation, which can momentarily have negative impacts on SQL service latency. Exclude these goroutine stacks by using the `--include-goroutine-stacks=false` flag. Note that any periodic goroutine dumps previously taken on the node will still be included in `nodes/*/goroutines/*.txt.gz``, as these would have already been generated and don't require any additional stop-the-world operations to be collected.<br /><br />**Default:** true
+`--include-range-info` | Include one file per node with information about the KV ranges stored on that node, in `nodes/{node ID}/ranges.json`.<br /><br />This information can be vital when debugging issues that involve the [KV storage layer]({% link {{ page.version.version }}/architecture/overview.md %}#layers), such as data placement, load balancing, performance or other behaviors. In certain situations, on large clusters with large numbers of ranges, these files can be omitted if and only if the issue being investigated is already known to be in another layer of the system (for example, an error message about an unsupported feature or incompatible value in a SQL schema change or statement). Note however many higher-level issues are ultimately related to the underlying KV storage layer described by these files so only set this to `false` if directed to do so by Cockroach Labs support.<br /><br />**Default:** true
```

Comment:
suggest changing 'KV storage layer' to say only 'KV layer' since (at least in the docs if not the code) it includes 'Storage layer' as a subset.

Could also clarify further by writing something like

>  KV layer (which includes everything below the SQL layer)